cmd/up/up.go

```diff
@@ -119,6 +120,8 @@ More information is available here: https://okteto.com/docs/reference/cli#up`)
 				return err
 			}
 
+			log.Init(logrus.WarnLevel, config.GetDeploymentHome(dev.Namespace, dev.Name), config.VersionString)
```

Comment:
Here, we should initialize the file logger in debug level

Reply:
Double check `dev.Namespace` is always initialized at this point. You might need to run:
```
dev.Namespace, _ = k8Client.GetNamespace(k8sContext)
```

Reply:
It's initialized on loadDevOverrides if its not setted in your Okteto manifest.

Reply:
I think if the user does not set the `namespace` flag and it is not defined in the okteto manifest, it is not initialized until here:
https://github.com/okteto/okteto/blob/master/cmd/up/up.go#L229
Maybe we can refactor that to always initialize it as part of `loadDevOverrides`